WebRoot/js/zimbraMail/share/model/ZmFolder.js

```diff
@@ -171,7 +166,6 @@ ZmFolder.TCON_CODE[ZmFolder.ID_OTHER]			= "o";
 
 // folders that look like mail folders that we don't want to show
 ZmFolder.HIDE_ID = {};
-ZmFolder.HIDE_ID[ZmOrganizer.ID_CHATS]				= true;
```

Comment:
This change made the Chats folder visible again in webmail, you may want to restore this one.